.github/workflows/stale-issues.yml

```diff
@@ -0,0 +1,30 @@
+name: Mark and Close Stale Issues
+
+on:
+  schedule:
+  - cron: 0 */12 * * * # Every 12 hours
+
+jobs:
+  staleIssues:
+    # Only run this on repositories in the 'spinnaker' org, not on forks.
+    if: startsWith(github.repository, 'spinnaker/')
+    runs-on: ubuntu-latest
+    steps:
+    - name: stale
+      uses: gatsbyjs/stale@master
+      with:
+        DRY_RUN: true
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
This will perform actions as 'github-actions' bot.  Should we use use spinnakerbot's auth token instead?

It's also worth noting that the github-actions bot will need to be added as an oss-approver if we stick with `GITHUB_TOKEN` (spinnakerbot is an oss-approver already)